src/assets/Generator.Shared/ResponseHeadersExtensions.cs

```diff
@@ -40,7 +68,16 @@ public static bool TryGetValue(this ResponseHeaders headers, string name, out Da
         {
             if (headers.TryGetValue(name, out string? stringValue))
             {
-                value = TypeFormatters.ParseDateTimeOffset(stringValue, "");
+                if (DateTimeOffset.TryParseExact(stringValue, "r", DateTimeFormatInfo.InvariantInfo, DateTimeStyles.None, out var dto) ||
```

Comment:
Do we have a sense of how broadly this change will take affect (which libraries/operations), and whether there are any possible breaking behaviors?

Reply:
It will affect most of the headers. There won't be any breaking behavior cause we fallback to the old code path if `TryParseExact` fails.

Reply:
Is it possible that a header could ever parse to a different value than it did previously?